kernel-ewasm/cap9-std/src/proc_table/mod.rs

```diff
@@ -463,7 +476,7 @@ pub mod contract {
         fn get_proc_cap_list_len(&mut self, key: String, cap_type: U256) -> U256;
 
         /// Get Procedure Capability by Id, Type and Index
-        fn get_proc_cap(&mut self, key: String, cap_type: U256, cap_index: U256) -> Vec<U256>;
+        // fn get_proc_cap(&mut self, key: String, cap_type: U256, cap_index: U256) -> Vec<U256>;
```

Comment:
This is commented out

Reply:
They should be removed.

Reply:
Addressed in af037b85b72dd39a4b0e4a0f847a89b3a1b01e04.